### DISCUSSION TITLE
How to use two variables in url?

### DISCUSSION THREAD

**Discussion by sergey-dme** [Budibase/budibase#1906] [Help] [answered]
Hey all. We often use `/:id` and `{{ URL.id }}` for it to get the id value from url. What about two variables, for example `{{ URL.id1 }}` and `{{ URL.id2 }}` Is it possible? What syntax should be in that case? `.../:id1&:id2` - Just tried it, but seems it's incorrect one.

**Comment by aptkingston** [comment, marked as answer]
Hey @sergey-dme. All the parameters need to be route parameters rather than query string parameters, so your screen should have a route of `/foo/:id1/:id2`. That will let you do exactly what you're after there. You can also even make the parameters optional by using a question mark, such as `.../:id2?`.

**Reply by sergey-dme** [reply]
@aptkingston Thank you 🙏 That's very useful information
